src/main/java/org/veupathdb/service/eda/ds/plugin/standalonemap/markers/ContinuousAggregators.java

```diff
@@ -0,0 +1,130 @@
+package org.veupathdb.service.eda.ds.plugin.standalonemap.markers;
+
+import java.util.Comparator;
+import java.util.PriorityQueue;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+/**
+ * Aggregators that collect variable values and produce some aggregated result.
+ */
+public enum ContinuousAggregators {
+  Mean(org.veupathdb.service.eda.generated.model.Aggregator.MEAN.getValue(), () -> new MarkerAggregator<Double>() {
+    private double sum = 0;
+    private int n = 0;
+
+    @Override
+    public void addValue(String d) {
+      if (d == null || d.isEmpty()) {
+        return;
+      }
+      sum += Double.parseDouble(d);
+      n += 1;
+    }
+
+    @Override
+    public Double finish() {
+      if (n == 0) {
+        return null;
+      }
+      return sum / n;
```

Comment:
how many digits will this report to? should we format or the client? i dont remember what we do other places..